pkg/kv/kvserver/store.go

```diff
@@ -2100,9 +2100,15 @@ func (s *Store) startGossip() {
 			interval:    s.cfg.SentinelGossipTTL() / 2,
 		},
 		{
-			key:         keys.SystemConfigSpan.Key,
```

Comment:
Do we need this anymore? We don't need to be gossiping writes to these two tables, right? IIUC, it was only there for compatibility with 21.2 versions, which this code (to-be-released in v22.2) doesn't care about.